system/inc/system_network.h

```diff
@@ -102,6 +102,15 @@ typedef WLanCredentials NetworkCredentials;
  * @return 0 on success.
  */
 int network_set_credentials(network_handle_t network, uint32_t flags, NetworkCredentials* creds, void* reserved);
+/**
+ *
+ * @param network   The network to configure the credentials.
+ * @param flags     Flags. set to 0.
+ * @param creds     The credentials to set. Should not be NULL.
```

Comment:
`creds` is not used in the function, and is normally set to NULL.  When clearing creds, this argument is N/A and should be set to NULL.

Reply:
ok, will change description